molo/core/content_import/helpers/parse.py

```diff
@@ -12,6 +12,10 @@ def parse_validate_content(repos, locales):
     }
 
 
+def parse_import_content(repos, locales):
+    return parse_validate_content(repos, locales)
```

Comment:
Its a case of making provision for possible changes later, in case the way import requests are parsed later on changes from the way validate requests are parsed.